google/cloud/internal/grpc_opentelemetry.h

```diff
@@ -93,6 +96,20 @@ future<T> EndSpan(
 
 #endif  // GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
 
+/**
+ * Returns a traced timer, if OpenTelemetry tracing is enabled.
+ */
+template <typename Rep, typename Period>
+future<StatusOr<std::chrono::system_clock::time_point>> TracedAsyncBackoff(
+    CompletionQueue& cq, std::chrono::duration<Rep, Period> duration) {
+#ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
+  if (TracingEnabled(CurrentOptions())) {
+    return EndSpan(MakeSpan("Async Backoff"), cq.MakeRelativeTimer(duration));
+  }
+#endif  // GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
+  return cq.MakeRelativeTimer(duration);
```

Comment:
I would be tempted to factor out the common `MakeRelativeTimer()` bits.

Reply:
Done, but you cost me a line of code.

Reply:
So then maybe that's timer = EndSpan(MakeSpan("Async Backoff"), std::move(timer));?

Reply:
Done.